cmd/cli/plugin/feature/go.mod

```diff
@@ -0,0 +1,93 @@
+module github.com/vmware-tanzu/tanzu-framework/featuregate/plugin
+
+go 1.17
+
+replace (
+	github.com/vmware-tanzu/tanzu-framework/apis/cli => ./../../../../apis/cli
+	github.com/vmware-tanzu/tanzu-framework/apis/config => ./../../../../apis/config
+	github.com/vmware-tanzu/tanzu-framework/apis/run => ./../../../../apis/run
+	github.com/vmware-tanzu/tanzu-framework/cli/runtime => ./../../../../cli/runtime
+	github.com/vmware-tanzu/tanzu-framework/featuregates/client => ./../../../../featuregates/client
+	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.1.5
+)
+
+require (
+	github.com/aunum/log v0.0.0-20200821225356-38d2e2c8b489
+	github.com/spf13/cobra v1.5.0
+	github.com/vmware-tanzu/tanzu-framework/apis/cli v0.0.0-00010101000000-000000000000
+	github.com/vmware-tanzu/tanzu-framework/apis/config v0.0.0-20220824221239-af5a644ffef7
```

Comment:
should the version for apis/config and cli/runtime be `v0.0.0-00010101000000-000000000000` ?

Reply:
I think `go mod tidy` updates that version automatically. In any case, we use `replace` so it uses right version anyway. We can look into this in a follow up PR.